pkg/services/sqlstore/sqlstore.go

```diff
@@ -238,10 +238,6 @@ func (ss *SqlStore) getEngine() (*xorm.Engine, error) {
 		return nil, err
 	}
 
-	if ss.Cfg.IsDatabaseMetricsEnabled() {
-		ss.dbCfg.Type = WrapDatabaseDriverWithHooks(ss.dbCfg.Type)
-	}
-
```

Comment:
Why change this file as well? The PR title gives the impression it will only touch the changelog?

Reply:
@marefr Do you know if this change makes sense?

Reply:
Yes we want to remove this temporarily so that users wont get a corrupted database.